scripts/setup/install_desktop_packages.sh

```diff
@@ -50,12 +50,12 @@ then
  
   if [ "${NO_TUNNEL}" -eq 1 ]; then
       echo "Getting the custom Debian without tunnel"
-      sudo /bin/bash -c "echo \"deb [arch=amd64] http://astrobee.ndc.nasa.gov/software ${DIST} main\" > $arssrc" || exit 1
-      sudo /bin/bash -c "echo \"deb-src http://astrobee.ndc.nasa.gov/software ${DIST} main\" >> $arssrc" || exit 1
+      sudo /bin/bash -c "echo \"deb [arch=amd64] http://astrobee.ndc.nasa.gov/software_new ${DIST} main\" > $arssrc" || exit 1
+      sudo /bin/bash -c "echo \"deb-src http://astrobee.ndc.nasa.gov/software_new ${DIST} main\" >> $arssrc" || exit 1
   else
       echo "Tunnelling to get the custom Debian"
-      sudo /bin/bash -c "echo \"deb [arch=amd64] http://127.0.0.1:8765/software ${DIST} main\" > $arssrc" || exit 1
-      sudo /bin/bash -c "echo \"deb-src http://127.0.0.1:8765/software ${DIST} main\" >> $arssrc" || exit 1
+      sudo /bin/bash -c "echo \"deb [arch=amd64] http://127.0.0.1:8765/software_new ${DIST} main\" > $arssrc" || exit 1
+      sudo /bin/bash -c "echo \"deb-src http://127.0.0.1:8765/software_new ${DIST} main\" >> $arssrc" || exit 1
       ssh -N -L 8765:astrobee.ndc.nasa.gov:80 ${username}m.ndc.nasa.gov &
```

Comment:
For security reasons when you use the `-L` argument to `ssh` you should always start it with `127.0.0.1:`, which will restrict the local end of the SSH tunnel to bind to the loopback interface only.

Otherwise by default it will bind to all network interfaces on the local host, which you probably won't need, but it will make the tunnel accessible from other hosts on your local network. The tunnel will show up in a port scan and you might get a warning from network security admins.

Reply:
Interesting observation. However, since this is not exactly related to this PR and it was not introduced here maybe it can go on its own PR. What do you think?

Actually, do we still use the tunnel to get our debians? Is m still reachable without VPN?

Reply:
Well my main point was to spread the word about the security issue. If you don't feel it makes sense to address it in this PR that's really up to you (I'm not trying to say it should be a blocker).

If it was me, I would probably add the `127.0.0.1:` prefix to immediately fix the security issue and also add a comment suggesting somebody check if the tunnel is still needed.

Reply:
Same here. We can keep the tunnel option for now just in case we cannot directly access the astrobee server for some reason. But maybe we should make NO_TUNNEL default?